R/load_epidata.R

```diff
@@ -69,10 +70,10 @@ load_epidata_raw <- function(pathogen, table = c("article", "parameter",
     ## parameters.
     cols <- intersect(colnames(tmp), names(col_types))
     col_types <- col_types[cols]
-    out <- read_csv(file_path, col_types = col_types, show_col_types = FALSE, col_select = colnames(tmp)) 
+    out <- read_csv(file_path, col_types = col_types, show_col_types = FALSE, col_select = colnames(tmp))
```

Comment:
I suspect the code would either break or return nothing here, because cols would be empty. We haven't defined col_types for our new file.